src/boot/efi-log.c

```diff
@@ -42,6 +42,10 @@ int log_set_max_level(LogLevel level) {
         return old;
 }
 
+LogLevel log_get_max_level(void) {
```

Comment:
I'd like to also move this above `log_set_max_level()`, to make the order consistent with the header, but not a matter.